project/Build.scala

```diff
@@ -2362,8 +2362,7 @@ object ScaladocConfigs {
         s"$dottyLibRoot/src/scala=compile",
         s"$dottyLibRoot/src/scala/compiletime=compile",
         s"$dottyLibRoot/src/scala/util=compile",
-        s"$dottyLibRoot/src/scala/util/control=compile",
-        "docs/_docs/reference/enums=compile"
+        s"$dottyLibRoot/src/scala/util/control=compile"
```

Comment:
Huh, I'm confused. Is this an inclusion list or an exclusion list?

Reply:
Thank you for feedback. I think it is an inclusion list and I did something weird 🤔 I'll fix it.

Reply:
Hmm, I have no idea why the previous contributor added files to `stableScala3` instead of `Scala3` task. I think it is enough to add them to `Scala3` task as `stableScala3` uses `Scala3` value anyway.

Reply:
If there was something specific to `stableScala3` task, it should have been added to `stableScala3`, but I don't think there is with regard to files to apply the snippet compiler.

Reply:
I'm not sure this change https://github.com/lampepfl/dotty/pull/19498/commits/f3cb10b5d6737fadfc65af73b34cf914148678cf is necessary, but in case `dottyLibRoot/src` is somehow different from `scalaLibrarySrc`, files under `dottyLibrarySrc` are also included to snippet compiler.